package/finit/finit.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-FINIT_VERSION = 4.9
+FINIT_VERSION = 4.10-rc1
```

Comment:
rc1, sounds dangerous.

Reply:
Current 4.9 plus patch backport is a lot worse.